packages/portal-proto/src/features/facets/NumericRangeFacet.tsx

```diff
@@ -741,7 +751,7 @@ const Year: React.FC<NumericFacetData> = ({
     <div className="flex flex-col w-100 space-y-2 px-2 mt-1 ">
       <RangeInputWithPrefixedRanges
         hooks={{ ...hooks }}
-        units="year"
+        units="years"
```

Comment:
I deleted this, but have determined that this is the cause of the issue @wteouchicago reported. There is a unit type **year**, which represents a calendar year, as opposed to **years**, which is converted to days.

Reply:
Reverted this